### ISSUE TITLE
Add SkyBox support

### ISSUE THREAD

**Issue by ArnaudCourbiere** [ArnaudCourbiere/Keola#3]
Update Keola's Renderer to support a SkyBox parameter (pass a skybox to render).
Blocked by #2

**Comment by ArnaudCourbiere** [comment]
For now, just rely on bi-linear sampling. Putting quality optimization in another issue.

**Comment by ArnaudCourbiere** [comment]
Creating test resource in TestRenderer (242b706)
Next: Pass skybox to actual renderer and update renderer to use it with the skybox pipeline.
